Objects/typeobject.c

```diff
@@ -2938,43 +2942,21 @@ PyType_GetSlot(PyTypeObject *type, int slot)
     return  *(void**)(((char*)type) + slotoffsets[slot]);
 }
 
-/* Internal API to look for a name through the MRO.
-   This returns a borrowed reference, and doesn't set an exception! */
-PyObject *
-_PyType_Lookup(PyTypeObject *type, PyObject *name)
+/* Internal API to look for a name through the MRO, bypassing the method cache.
+   This returns a borrowed reference, and might set an exception! */
+static PyObject *
+_PyType_LookupUncached(PyTypeObject *type, PyObject *name, int *error)
```

Comment:
Local functions usually use different name convention. All lower characters and no `Py` prefix.

Reply:
I was trying to hint at the name of the existing function. A different name would be less clear, I think.

Reply:
IMHO this rather adds a confusion.

Reply:
I renamed the function.